packages/frontend/src/components/campaigns/campaign/index.tsx

```diff
@@ -1,39 +1,43 @@
-import type { AmmInfo } from "@/src/types";
 import { Apr, SkeletonApr } from "./apr";
 import { Pool, SkeletonPool } from "./pool";
 import { SkeletonStatus, Status } from "./status";
 import { Rewards, SkeletonRewards } from "./rewards";
 import { Chain, SkeletonChain } from "./chain";
 import type { NamedCampaign } from "@/src/hooks/useCampaigns";
 
+import styles from "./styles.module.css";
+import Link from "next/link";
```

Comment:
Import here.

Reply:
Addressed in e3a2263d1d5914abd841a5695dcc92f0d2856e17